### PR TITLE
Added Hexabin API

### PR DESCRIPTION
**Pull request by angelina-tsuboi**
- [ ] My submission is formatted according to the guidelines in the [contributing guide](CONTRIBUTING.md)
- [ ] My addition is ordered alphabetically
- [ ] My submission has a useful description
- [ ] The description does not end with punctuation
- [ ] Each table column is padded with one space on either side
- [ ] I have searched the repository for any relevant issues or pull requests
- [ ] Any category I am creating has the minimum requirement of 3 items
- [ ] All changes have been [squashed][squash-link] into a single commit

[squash-link]: <https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit>

### CODE REVIEW

**Review by marekdano — CHANGES_REQUESTED** [review]
Thanks for the contribution @angelina-tsuboi!
I left a comment below.

File: README.md [hunk 1]
```diff
@@ -260,6 +260,7 @@ API | Description | Auth | HTTPS | CORS |
 | [GitHub](https://docs.github.com/en/free-pro-team@latest/rest) | Make use of GitHub repositories, code and user info programmatically | `OAuth` | Yes | Yes |
 | [Gitlab](https://docs.gitlab.com/ee/api/) | Automate GitLab interaction programmatically | `OAuth` | Yes | Unknown |
 | [Gitter](https://developer.gitter.im/docs/welcome) | Chat for Developers | `OAuth` | Yes | Unknown |
+| [Hexabin](http://hexabin-api.com/) | Convert and retrieve hexadecimal, binary, decimal, and octal values with ease | No | No | Unknown |
```
[the remark this answers is not in the document]
Comment (angelina-tsuboi) [reply README.md]: Hi @marekdano I just updated the link. Thank you so much!

**Comment by marekdano** [comment]
@angelina-tsuboi thanks for the update!

Can you please resolve the conflicts? 
We won't be able to merge your changes till the conflict is resolved. Or if it's easier close this PR and open a new one with your record.

**Comment by angelina-tsuboi** [comment]
@marekdano Thanks for letting me know! I resolved the conflicts.

File: README.md [hunk 2]
```diff
@@ -277,7 +277,7 @@ API | Description | Auth | HTTPS | CORS |
 | [GitHub](https://docs.github.com/en/free-pro-team@latest/rest) | Make use of GitHub repositories, code and user info programmatically | `OAuth` | Yes | Yes |
 | [Gitlab](https://docs.gitlab.com/ee/api/) | Automate GitLab interaction programmatically | `OAuth` | Yes | Unknown |
 | [Gitter](https://developer.gitter.im/docs/welcome) | Chat for Developers | `OAuth` | Yes | Unknown |
-| [Host.io](https://host.io) | Domains Data API for Developers | `apiKey` | Yes | Yes |
```
Comment (marekdano) [inline README.md:280]: Thanks for resolving the conflict! 
Can I ask you to keep this record in README, please? It should be after your Hexabin API record.

**Review by marekdano — CHANGES_REQUESTED** [review]
Please keep the removed Host.io record in the README.

File: README.md (hunk 2 above)
Comment (angelina-tsuboi) [reply README.md:280]: @marekdano Yes, I just added the record. Thank you!

**Review by marekdano — APPROVED** [review]

**Pull request merged**